services/jenkins/jenkins.tester.js

```diff
@@ -40,3 +40,40 @@ t.create('connection error')
   .get('/plugin/v/blueocean.json')
   .networkOff()
   .expectJSON({ name: 'plugin', value: 'inaccessible' });
+
```

Comment:
Could you please specify "cobertura: " at the beginning of the previous test names?